File: PrenotazioniVacanze/src/app/dati/utente-login.ts
export class UtenteLogin {
  email : string;
  passwordUtente : string;
  constructor(email:string, passwordUtente:string){
    this.email=email;
    this.passwordUtente=passwordUtente;
  }
  public getEmail() : string {
    return this.email;
  }
  public setEmail(email: string) {
    this.email = email;
  }
  public getPasswordUtente() : string {
    return this.passwordUtente;
  }
  public setPasswordUtente( passwordUtente : string) {
    this.passwordUtente = passwordUtente;
  }
}

File: PrenotazioniVacanzeSpring/src/main/java/com/example/PrenotazioniVacanzeSpring/UtenteLogin.java
package com.example.PrenotazioniVacanzeSpring;

public class UtenteLogin {
	private String email;
	private String passwordUtente;
	public UtenteLogin() {}
	
	public UtenteLogin(String email, String passwordUtente) {
		super();
		this.email = email;
		this.passwordUtente = passwordUtente;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getPasswordUtente() {
		return passwordUtente;
	}
	public void setPasswordUtente(String passwordUtente) {
		this.passwordUtente = passwordUtente;
	}
	

}
